README.md

```diff
@@ -124,19 +124,19 @@ of routing the traffic from Zipkin libraries to the Jaeger backend.
 
 ## Version Compatibility Guarantees
 
-Occasionally, CLI flags can be deprecated due to, for example, usability improvements or new functionality.
+Since Jaeger uses many components from the [OpenTelemetry Collector](https://github.com/open-telemetry/opentelemetry-collector/) we try to maintain configuration compatibility between Jaeger releases. Occasionally, configuration elements in Jaeger (or in Jaeger v1 CLI flags) can be deprecated due to usability improvements, new functionality, or changes in our dependencies.
```

Comment:
I think "configuration option" used in L129 sounds better than "element", perhaps converge to that one for consistency?

Reply:
Should be resolved in latest push